### PR TITLE
Fixed footer, navbar, created files reset.css updated .stylelintrc.json

### PR DESCRIPTION
**Pull request by Mariion-62**

### CODE REVIEW

**Review by Um-Brehon — DISMISSED** [review]
Saw no evil there (I really didn't 😇)

File: src/Components/Collection/index.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 function Collection() {
   return (
     <div>
-      <p>Collection</p>
+      <p>guvibgh vnhrnfuvoij nhuvrunvj nhuvgirr</p>
```
Comment (Hacene010) [inline src/Components/Collection/index.jsx]: This souldn't be here. plz put something else. (ex: name of the component)

File: src/Components/Game/index.jsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 function Game() {
   return (
     <div>
-      <p>Game</p>
+      <p>esdrfgtyh gyuhujio ug h</p>
```
Comment (Hacene010) [inline src/Components/Game/index.jsx]: same here

File: src/Components/Home/index.jsx [hunk 3]
```diff
@@ -1,9 +1,10 @@
 import React from 'react';
+import './index.css';
 
 function Home() {
   return (
-    <div>
-      <p>Home</p>
+    <div className="home">
+      <p>lirhyelkigufliejlkeji</p>
```
Comment (Hacene010) [inline src/Components/Home/index.jsx]: same here

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Some changes to make

**Review by Um-Brehon — APPROVED** [review]

**Review by Hacene010 — APPROVED** [review]
that's better ; )

**Pull request merged**